### ISSUE TITLE
Always: few Days after Installation, Database Problem

### ISSUE THREAD

**Issue by Hirnsausen** [Drive4ik/simple-tab-groups#283]
**Describe the bug**
It happens too often: each time I install STG, a few days or weeks later I get some STG database error and have to remove and re-install STG. Each time therefore I lose all my groups.

**To Reproduce**
1. I use Firefox Nightly, latest version
2. I install STG and create groups filled with tabs
3. I use STG for some days to weeks
4. CRASH! BOUM! BANG! Database Error Again!

**Expected behavior**
I expected to happen, that STG would continue to function properly.

**Screenshots**
![firefox - 02](https://user-images.githubusercontent.com/16428907/47923100-8699a400-de86-11e8-9ac9-a3761354bbc6.jpg)

**Desktop (please complete the following information):**
 - Windows 7 Home Premium 64bit
 - Firefox Nightly (always updated to the latest version)
 - Simple Tab Groups version: I downloaded the STG two weeks ago

**Additional context**
(n/a)

**Comment by Drive4ik** [comment]
Firefox nightly is not a stable build, and you should not expect it to work properly. What version of STG are you using? Versions 3.3+ make automatic data backup. Look in the addon settings.

**Comment by Hirnsausen** [comment]
Hi, the last STG version was 3.2.1, but upon your hint I downloaded and installed the latest STG 3.3 version. Thanks for that hint. I have to create all groups new again, but that was expected.

To be on the save side, where would that STG database be? I want to make a manual backup, too, after having created and filled the groups again.

**Comment by Hirnsausen** [comment]
And there is another bad problem (I am not the only affected person):
the "forgetting" of tab contents upon Firefox start. 

At my company, we have many important tabs open, but it happens quite often, that when Firefox has started, the one or other tab appear empty, their former website address being "forgotten".  We have too many tabs, so our staff cannot remember which address was open. Happened with the frweshly installed latest version 3.3, too.

**Comment by Drive4ik** [comment]
Firefox may not save the url tabs if this tab is loaded when you close Firefox. You can test it on the new Firefox profile (without add-ons) - open the heavy tab and close Firefox without waiting for it to load.

**Comment by Drive4ik** [comment]
Can you retest this issue with latest STG? (3.4.2.1 or higher)

**Comment by tonycpsu** [comment]
I'm now getting this error with Simple Tab Groups 3.4.4 on Firefox 66.0.3/MacOS.

![image](https://user-images.githubusercontent.com/914142/56918396-98d15500-6a59-11e9-8d56-67425bb799bf.png)

**Comment by Drive4ik** [comment]
@tonycpsu Yes, remove addon, restart browser, install addon, restore backup
